x-pack/plugins/security_solution/public/flyout/document_details/shared/utils/highlighted_fields_helpers.ts

```diff
@@ -16,7 +16,8 @@ import type { HighlightedFieldsTableRow } from '../../right/components/highlight
  */
 export const convertHighlightedFieldsToTableRow = (
   highlightedFields: UseHighlightedFieldsResult,
-  scopeId: string
+  scopeId: string,
+  isReadOnly: boolean
```

Comment:
consider keeping `isPreview` here and just do isReadOnly: isPreview` down below, for the same reason as mentioned above in the `footer.tsx` file